apps/shell/test/allowlist.test.js

```diff
@@ -26,3 +78,22 @@ describe('Helper function hostInAllowList()', () => {
     expect(helpers.hostInAllowList(ALLOW_LIST, 'p1001.eleven.osc.edu')).not.toBeTruthy()
   })
 })
+
+describe('Helper function host_and_dir_from_url', () => {
```

Comment:
I think all of the functions in here are helpers. `describe(host_and_dir_from_url)` is sufficient in my opinion and more more to the point.